test-fuzz/tests/integration/ci.rs

```diff
@@ -28,6 +30,22 @@ fn dylint() {
         .success();
 }
 
+#[test]
+fn cargo_sort() {
+    for entry in WalkDir::new("..")
+        .into_iter()
+        .filter_map(Result::ok)
+        .filter(|e| e.file_name() == (OsStr::new("Cargo.toml")))
```

Comment:
```suggestion
        .filter(|e| e.file_name() == OsStr::new("Cargo.toml"))
```
TIL `walk_dir::DirEntry::file_name` returns a `&OsStr`.

Could you remove these extra parentheses and squash your commits down to one?